.github/workflows/website-deploy.yml

```
@@ -0,0 +1,20 @@
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Set up AWS CLI
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: ${{ secrets.AWS_REGION }}
+      - name: Trigger CodePipeline
+        run: |
+          aws codepipeline start-pipeline-execution --name  ci-cd-website-prod-pipeline
```

yamllint (1.29.0-1)

[error] no new line character at the end of file

(new-line-at-end-of-file)